src/FakeItEasy/FakeItEasy.csproj

```diff
@@ -63,6 +63,8 @@
     <Compile Include="AsyncReturnValueConfigurationExtensions.cs" />
     <Compile Include="Compatibility\MethodInfoWrapper.cs" />
     <Compile Include="Configuration\PropertyExpressionHelper.cs" />
+    <Compile Include="Creation\CastleDynamicProxy\DynamicProxyResources.cs" />

```

Comment:
Ah, I thought I had... I used R#'s rename command, which normally renames the file, but it didn't work in this case.